### ISSUE TITLE
CR Request for CSS Conditional Rules Level 4 (CSSWG)

### ISSUE THREAD

**Issue by fantasai** [w3c/transitions#406]
# Document title, URLs, estimated publication date
CSS Conditional Rules Module Level 4
https://www.w3.org/TR/css-conditional-4/
January?

# Abstract
https://www.w3.org/TR/css-conditional-4/#abstract

# Status
https://www.w3.org/TR/css-conditional-4/#sotd

# Link to group's decision to request transition
https://lists.w3.org/Archives/Public/www-style/2021Dec/0009.html

# Changes
https://www.w3.org/TR/css-conditional-4/#changes-from-20200303

# Requirements satisfied
afaict

# Dependencies met (or not)
yes

# Wide Review
Straightforward minor feature addition to Level 3's supports query syntax; shipping in most browsers for more than a year. https://caniuse.com/?search=selector()
We could issue horizontal review requests, but I can't think of any reason it would impact any of the horizontal review groups' concerns, so I think it would be a waste of their time that would be better spent elsewhere. If in the Director's judgement there's a reason to request formal review, let me know and I'll issue the relevant requests.

See https://github.com/w3c/transitions/issues/404 for all Level 3 features

# Issues addressed
https://github.com/w3c/csswg-drafts/issues/3207
https://github.com/w3c/csswg-drafts/issues/3936
Basically, just addressed the one issue requesting the feature, and one issue which will likely be closed as already addressed, and if not, would be a new feature in Level 5.

# Formal Objections
None

# Implementation
https://caniuse.com/?search=selector()

# Patent disclosures
None

**Comment by swickr** [comment]
Please do request Privacy review; while the assertions in the Privacy Considerations section may be true, PING should be given the opportunity to review this text.

As currently written (Introduction Issue 1) this is a [diffspec](https://www.w3.org/2003/12/22-pp-faq#diffspec).  If the Working Group intends to copy the level 3 features into this spec -- which we prefer per [PP FAQ #28](https://www.w3.org/2003/12/22-pp-faq#diffspec) -- then an updated CR will be needed in order to issue a new Call for Exclusion.

**Comment by svgeesus** [comment]
Privacy review complete, [no issues found](https://github.com/w3cping/privacy-request/issues/70#issuecomment-1017797960)

**Comment by swickr** [comment]
Transition approved.

**Comment by svgeesus** [comment]
[pubrules checker has been borked](https://www.w3.org/pubrules/?url=https%3A%2F%2Fwww.w3.org%2FTR%2F2022%2FCR-css-conditional-4-20220217%2F&profile=CR&validation=simple-validation&informativeOnly=false&echidnaReady=false&patentPolicy=pp2020) all day, will try again tomorrow

**Comment by svgeesus** [comment]
Publication requested 14 Feb, expected 17 Feb

**Comment by svgeesus** [comment]
[Published 17 Feb](https://www.w3.org/TR/2022/CR-css-conditional-4-20220217/)
